plugins/obs-ffmpeg/obs-ffmpeg-nvenc.c

```diff
@@ -292,6 +293,14 @@ static void on_first_packet(void *data, AVPacket *pkt, struct darray *da)
 					&enc->sei.array, &enc->sei.num);
 	}
 	da->capacity = da->num;
+
+	if (enc->ffve.context->max_b_frames != 0) {
+		int64_t expected_dts = -(enc->ffve.context->max_b_frames *
+					 enc->ffve.context->time_base.num);
+		if (pkt->dts != expected_dts) // Unpatched FFmpeg
+			enc->dts_offset =
```

Comment:
Could you add a `LOG_WARNING` message here to indicate that the workaround is being applied?